src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -496,5 +499,48 @@ public static void UnixCannotModifyDisallowedStore(bool useEnum, OpenFlags openF
                 Assert.Equal(0, store.Certificates.Count);
             }
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux)] // Windows/OSX doesn't use SSL_CERT_{DIR,FILE}.
```

Comment:
@stephentoub do you want the OuterLoop attribute on this test?

Reply:
As long as everything successfully gets cleaned up, it's fine as inner loop.